istio/assets/monitors/failed_sidecar_injection.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "[Istio] Increased number of failed sidecar injection requests",
+	"type": "metric alert",
+	"query": "sum(last_5m):avg:istio.sidecar_injection.requests_total{*}.as_count() - avg:istio.sidecar_injection.success_total{*}.as_count() > 3",
```

Comment:
Any reason for `> 3` in particular?

Reply:
Just a suggestion, when a user is clicks into a recommended monitor, they can modify the metric thresholds